articles/introduction-to-wireless-networking/index.md

```diff
@@ -0,0 +1,122 @@
+### Introduction
+
+Wireless networks have exploded the world currently, all the same, we all hate wires all over, or don't we? Or perhaps not!  
+It has even replaced desktops that had connectors (RJ-45) on the wall.  
+Okay, let's say you go to a big hotel or airport, they will have a wireless network that you can use to access the internet.  
+The world has undergone a lot of internet evolution. 
+I would tell you this for sure if you wanted to buy a laptop that had the specification of a desktop with an RJ-45 connector on the wall,
+you would pay the double price.  
+Nowadays, the laptop price is affordable, or isn't it?  
+
```

Comment:
Pay attention to the formatting.

![image](https://user-images.githubusercontent.com/26842636/107795096-43754d80-6d7e-11eb-83e2-ddcfe6ba2d95.png)

It looks odd if each sentence occupies a line each.

Reply:
Make the change throughout the article.